docs/install_Device_Northbound.md

```diff
@@ -2,27 +2,27 @@
 
 - [Configuration Steps](#configuration-steps)
 - [Configure Northbound](#configure-northbound)
-  - [Databus](#databus)
-  - [External Databus](#external-databus)
-  - [IIH Insights Hub Sync](#iih-insights-hub-sync)
-    - [Integrate Data Service](#integrate-data-service)
+  - [Create Databus Credentials and Topics](#create-databus-credentials-and-topics)
+  - [Configure Communication with DataXess](#configure-communication-with-dataxess)
+  - [Insights Hub Sync](#iih-insights-hub-sync)
+    - [Integrate IIH Essentials](#integrate-iih-essentials)
     - [Configure Connections](#configure-connections)
     - [Create the Asset Model](#create-the-asset-model)
   - [Northbound Device - Energy Manager](#northbound-device---energy-manager)
 - [Navigation](#navigation)
   
 # Configure Northbound
 
-The Northbound consist of one device. In the following this is called "Central Device".
+The Northbound consist of one device. In the following this is called "Shopfloor-to-Cloud".
 Installed Apps on Central Device:
```

Comment:
Installed Apps are:

As you are not using Central Device anymore but renamed it, remove it from here

Reply:
Done